.github/workflows/ci-standalone.yml

```diff
@@ -0,0 +1,75 @@
+name: mycli
+
+on:
+  push:
+    branches:
+      - pasenor/pyoxidizer
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Cache cargo registry
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-cargo-registry
+
+      - name: Build with pyoxidizer
+        run: |
+          cargo install pyoxidizer
```

Comment:
just FYI, pyoxidizer can be installed via pip now: https://github.com/laixintao/iredis/pull/384
so we can drop cargo cache.